src/EnergyPlus/OutsideEnergySources.cc

```diff
@@ -560,7 +553,7 @@ void OutsideEnergySourceSpecs::oneTimeInit_new(EnergyPlusData &state)
         meterTypeKey = "DistrictHeatingSteam";
     }
     SetupOutputVariable(state,
-                        reportVarPrefix + "Energy",
+                        format("{}Energy", reportVarPrefix),
```

Comment:
Although format might not be saving a lot here since it was just one concatenation, it's still a good move.